microsoftgraph/microsoft-graph-explorer-v4
# Task: Add Adaptive card templates to local repo

## Overview

Fixes #2170 
Having them locally is best as the site isn't always maintained.

## Notes
- The templates are from the [Adaptive cards templates repo](https://github.com/microsoft/adaptivecards-templates).
- Commit history is preserved.

## Testing Instructions

* Launch GE
* Run GET /me query
* Click in Adaptive cards Pivot
* Observe card and JSON template are available
